src/main/java/net/sf/jabref/gui/JabRefFrame.java

```diff
@@ -483,6 +483,7 @@ public void actionPerformed(ActionEvent e) {
     private final List<Object> openDatabaseOnlyActions = new LinkedList<>();
     private final List<Object> severalDatabasesOnlyActions = new LinkedList<>();
     private final List<Object> openAndSavedDatabasesOnlyActions = new LinkedList<>();
+    private final List<Object> remoteDatabasesOnlyActions = new LinkedList<>();
```

Comment:
Those lists are used to add actions. Unfortunately there are differnt types of them, such as `AbstractAction`, `GeneralActions` which extends `MnemonicAwareAction` just the interface `Action`.
I think this could be a good stuff for a new PR.